extensions/wikia/Chat2/ChatBanListSpecial_ajax.php

```diff
@@ -0,0 +1,74 @@
+<?php
+
+/**
+ * @package MediaWiki
+ * @subpackage SpecialPage
+ * @author Piotr Molski <moli@wikia.com>
+ * @version: $Id$
+ */
+class ChatBanListAjax
```

Comment:
Instead of using the legacy deprecated MW AJAX entrypoint I think it could probably be better to make this a separate method of the `WikiaSpecialPageController` subclass, and use the Nirvana API  - also makes caching easier!